cmd/crowdsec-cli/install.go

```diff
@@ -63,6 +63,10 @@ you should [update cscli](./cscli_update.md).
 			if !config.configured {
 				return fmt.Errorf("you must configure cli before interacting with hub")
 			}
+
+			if err := setHubBranch(); err != nil {
+				log.Errorf("error while setting hub branch: %s", err)
```

Comment:
I think we want to return the error here, so that the `PersistentPreRunE` can properly fail